fdbclient/ServerKnobs.cpp

```diff
@@ -800,7 +800,9 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( CHECKPOINT_TRANSFER_BLOCK_BYTES,                      40e6 );
 	init( QUICK_GET_VALUE_FALLBACK,                             true );
 	init( QUICK_GET_KEY_VALUES_FALLBACK,                        true );
-	init( MAX_PARALLEL_QUICK_GET_VALUE,                           50 ); if ( randomize && BUGGIFY ) MAX_PARALLEL_QUICK_GET_VALUE = deterministicRandom()->randomInt(1, 100);
+	init( STRICTLY_ENFORCE_BYTE_LIMIT, 							false); if( randomize && BUGGIFY ) STRICTLY_ENFORCE_BYTE_LIMIT = deterministicRandom()->coinflip() ? false : true;
```

Comment:
nit, it's better to make it aligned with other knobs.

Reply:
it is aligned in my IDE, and also called clang-format, not sure what is going on here..

Reply:
`clang-format` was turned off for this block.

Reply:
I see, will fix

Reply:
https://github.com/apple/foundationdb/pull/8800